lib/buildpackutil.py

```diff
@@ -53,6 +53,18 @@ def get_database_config(development_mode=False):
         'DatabaseHost': match.group('host'),
         'DatabaseName': match.group('dbname'),
     }
+
+    if 'extra' in match.groupdict() and match.group('extra'):
+        extra = match.group('extra').lstrip('?')
+        import urlparse
```

Comment:
Shouldn't `import urlparse` be at the top of the file, along with the rest of the imports?

Additionally, if this project is python3, we'll want to use `urllib.parse` instead of `urlparse`.